### DISCUSSION TITLE
Error furo_asset_hash is undefined

### DISCUSSION THREAD

**Discussion by haruka-7** [pradyunsg/furo#314] [Potential Issue] [answered]
Hi,

First of all, thanks for the work done to give us this sphinx theme.

I have the error below when I try to build my doc : 

```
Reason: UndefinedError("'furo_asset_hash' is undefined")
make: *** [Makefile:20: html] Error 2
```

The full log : 

```
Running Sphinx v4.3.0
loading translations [fr]... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 48 source files that are out of date
updating environment: [new config] 48 added, 0 changed, 0 removed
reading sources... [100%] index                                                                               
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  2%] docs/contribute                                                                      
Theme error:
An error happened in rendering the page docs/contribute.
Reason: UndefinedError("'furo_asset_hash' is undefined")
make: *** [Makefile:20: html] Error 2
```
I'm using Debian 11.1, and I have install sphinx and furo from pip (python 3) : `pip install -U sphinx furo`

The `conf.py` I'm using is available here : https://github.com/pluxml/PluXml-Wiki/blob/migrationsphinx/conf.py

**Comment by pradyunsg** [comment, marked as answer]
Hiya! 👋🏽 

The problem is this line: https://github.com/pluxml/PluXml-Wiki/blob/cd892036defafcf50cd40f2b1a25c7a2890fb0fa/_templates/base.html#L101 -- there is no `furo_asset_hash` available to the template (as the error suggests).

See also https://pradyunsg.me/furo/stability/ :)

**Reply by haruka-7** [reply]
Hi, it's solved thanks for your help ;-)
